core/src/test/java/org/apache/iceberg/TestMetadataTableScans.java

```diff
@@ -1753,4 +1753,35 @@ public void testPositionDeletesManyColumns() {
     assertThat(scanTasks.get(0).file().path().toString()).isEqualTo("/path/to/delete1.parquet");
     assertThat(scanTasks.get(1).file().path().toString()).isEqualTo("/path/to/delete2.parquet");
   }
+
+  @TestTemplate
+  public void testFilesTableEstimateSize() throws Exception {
+    preparePartitionedTable(true);
+
+    assertEstimatedSize(new DataFilesTable(table));
+    assertEstimatedSize(new AllDataFilesTable(table));
+    assertEstimatedSize(new AllFilesTable(table));
+
+    if (formatVersion != 2) {
+      assertEstimatedSize(new DeleteFilesTable(table));
+      assertEstimatedSize(new AllDeleteFilesTable(table));
+    }
```

Comment:
Maybe misunderstanding something but shouldn't this be `if (formatVersion == 2)` for testing the delete files metadata tables?

Reply:
You are right, typo, i think there are no tasks so assert passed.  Let me rewrite it.